### ISSUE TITLE
Test case breaks due to unsupported methods on Linux

### ISSUE THREAD

**Issue by segabor** [mxcl/swift-sh#11]
`swift test` command halts with error message
```
Test Case 'IntegrationTests.testConventional' started at 2019-01-14 16:46:47.176
Fatal error: init(for:) is not yet implemented: file Foundation/Bundle.swift, line 57
```

**Comment by segabor** [comment]
This is not going to be implemented anytime soon. So we need to find another way to look up `swift-sh`.

**Comment by segabor** [comment]
Another missing method found
```
Test Case 'IntegrationTests.testConventional' started at 2019-01-14 17:02:42.165
Fatal error: url(for:in:appropriateFor:create:) is not yet implemented: file Foundation/FileManager.swift, line 134
```

**Comment by mxcl** [comment]
In #8 I'm using:

    return Path.root.join(#file).parent.parent.parent.join(".build/debug/swift-sh").string

**Comment by mxcl** [comment]
Fixed by #8, new release tagged and can be installed with any of the support methods.
